core/src/test/java/brooklyn/entity/rebind/RebindFeedTest.java

```diff
@@ -180,7 +180,7 @@ public void testSshFeedRegisteredInStartIsPersisted() throws Exception {
         EntityTestUtils.assertAttributeEqualsEventually(newEntity, SENSOR_INT, (Integer)0);
     }
 
-    @Test
+    @Test(groups="WIP") // intermittent failures
```

Comment:
Makes sense for the release, but I'd prefer if we get it back post-release (so it can nag us until fixed :) )